### DISCUSSION TITLE
New icon for scss

### DISCUSSION THREAD

**Discussion by BolotnikovMS** [zed-industries/zed#15768] [Feature Requests]
Hi, can you please update the icon for scss the files. 🙏
![file_type_scss_icon_130177](https://github.com/user-attachments/assets/08c7fc72-4f59-4d3d-8902-1ae45139e456)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
